app/hotels/src/filter/FilterStripe.js

```diff
@@ -31,13 +34,20 @@ type Props = {|
   filter: FilterParams,
   currency: string,
   currentSearchStats: CurrentSearchStats,
+  activeFilters: ActiveFilters,
 |};
 
 /**
  * This filter holds all available hotel filters. Active (selected) filters are
  * rendered first.
  */
 function FilterStripe(props: Props) {
+  const {
+    isPriceFilterActive,
+    isStarsFilterActive,
+    isMinScoreActive,
+    isHotelFacilitiesActive,
+  } = props.activeFilters;
```

Comment:
I am not super big fan of doing this. It's IMHO not very well readable. Example:

```js
<StarsFilter
    key="stars"
    stars={starsRating}
    onChange={props.onChange}
    isActive={isStarsFilterActive}
/>
```

Based on what should I know what is `starsRating` and `isStarsFilterActive`? This information is lost somewhere at the beginning of this component.